cgd_tx_eff/src/edu/ohsu/compbio/annovar/annovar_parser.py

```diff
@@ -422,14 +430,14 @@ def parse_file(self, annovar_file_type: AnnovarFileType, file_name:str, delimite
             for row in reader:
                 if annovar_file_type == AnnovarFileType.ExonicVariantFunction:
                     assert row[0].startswith('line')
-                    if len(row) != 18:
-                        self.logger.info(f"Exonic variant function file is expected have 18 columns. Found {len(row)}. Make sure you run the annotate_variation.pl script with the '--otherinfo' parameter to include other information from the original VCF")
+                    if len(row) != 9:
```

Comment:
the `convert2annovar.p` script appended each row of the vcf to each corresponding row of the avinput file and that's why we had 18 columns. Now all we have is the original avinput fields plus one more where i put the genomic variant id.